src/Entrust/Traits/EntrustRoleTrait.php

```diff
@@ -64,8 +66,7 @@ public function restore()
      */
     public function users()
     {
-        return $this->belongsToMany(Config::get('auth.providers.users.model'), Config::get('entrust.role_user_table'),Config::get('entrust.role_foreign_key'),Config::get('entrust.user_foreign_key'));
-       // return $this->belongsToMany(Config::get('auth.model'), Config::get('entrust.role_user_table'));
```

Comment:
We have version control so there is no need to leave commented out code in the codebase. People will just think it might be important in some way or the other while all it does is cluttering up the code.